triton-client/src/components/ProjectCardList/index.tsx

```diff
@@ -1,26 +1,29 @@
 import "@components/Common.scss"
 import { useNavigate } from "react-router-dom"
-import { Button, Typography } from "antd"
+import { List, Typography, Avatar } from "antd"
 import { ProjectCard, ProjectCardListProps } from "./interfaces"
+import projectIcon from "@static/project-icon.png"
 
 export default function ProjectCardList({ projects }: ProjectCardListProps) {
     const navigate = useNavigate()
-    const { Title } = Typography
+    const { Title, Text } = Typography
+    const items = projects
     return (
-        <>
-            <Title level={2}>Projects</Title>
-            {projects.map((project: ProjectCard) => (
-                <Button
-                    key={project.external_id}
-                    block
-                    onClick={() => navigate(`/project/${project.external_id}/`)}
-                    style={{ width: "90%", margin: "0 1rem" }}
+        <List
+            size="large"
+            header={<Title level={3}>Projects</Title>}
+            bordered
+            dataSource={items}
+            renderItem={(item: ProjectCard) => (
+                <List.Item
+                    onClick={() => navigate(`/project/${item.external_id}/`)}
                 >
-                    <Typography.Text strong>
-                        {project.external_name}
-                    </Typography.Text>
-                </Button>
-            ))}
-        </>
+                    <List.Item.Meta
+                        avatar={<Avatar src={projectIcon} />}
+                        title={<Text>{item.external_name}</Text>}
```

Comment:
When the project list gets collapsed, the words get wrapped letter by letter which can cause a card to grow vertically when collapsing project list.
I think we should avoid changing the width of the card while collapsing.

Reply:
![image](https://github.com/user-attachments/assets/efefc330-d297-4fc8-a060-37102cfad380)

Reply:
That picture is the last few frames before the list is completely collapsed.

Reply:
The same goes for the "Projects"

Reply:
the width of the menu has to change for the collapse to work.

Reply:
> the width of the menu has to change for the collapse to work.

Not necessarily. One strategy I can think of is controlling the overflow variable in css. Basically, allow elements in the sidebar to overflow but without showing horizontal scrollbar.